integreat_cms/cms/templates/organizations/member_list_row.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+
+<tr class="border-b border-solid border-gray-200">
+    <td class="text-sm text-left py-3 pl-4 pr-2">
+            {{ member.username }}
+    </td>
+    
+    <td class="text-sm text-left py-3 pl-4 pr-2">
+        {% if member.role %}
+            {{ member.role }}
+        {% else %}
+          - 
+        {% endif %}
```

Comment:
```suggestion
        {{ member.role }}
```
It's not possible to create users without a role anyway

Reply:
fixed